include/roboteam_ai/utilities/RobotDealer.h

```diff
@@ -10,6 +10,7 @@
 #include <set>
 #include <mutex>
 #include <vector>
+#include <include/roboteam_ai/world/Field.h>
```

Comment:
consistency in includes

<include/roboteam_ai/world/Field.h>
vs
"world/Field.h"

Reply:
Can we decide on one include convention within roboteam? :eyes: Not sure how enforcable it is but clang-format is already a good step in the good direction (Thanks emiel <3 )

Reply:
Agreed. There's probably something for this. I'll make it an issue.